### PR TITLE
Format all XML resources

### PR DESCRIPTION
**Pull request by wb9688**
#### What is it?
- [ ] Bug fix (user facing)
- [ ] Feature (user facing)
- [x] Code base improvement (dev facing)
- [ ] Meta improvement to the project (dev facing)

#### Description of the changes in your PR
Format all XML resources using Ctrl+Alt+L in Android Studio. I've also manually added newlines at the end of XML resources that currently didn't have a newline.

#### Fixes the following issue(s)
Seeing stuff like https://github.com/TeamNewPipe/NewPipe/pull/4453/files#diff-8093b831702ba14428d2d99c9edded66 in PRs, which makes it harder to review.

#### Agreement
- [x] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.

### CODE REVIEW

**Comment by avently** [comment]
Finally. THANK YOU.

**Comment by connectety** [comment]
Nice work.
IMO this PR should add xml-linting to make sure the work will not be undone in like a year.
I mean this can always be checked in the review, but that will often mean an extra commit and might be just forgotten.

**Comment by wb9688** [comment]
@connectety: That was also what I was thinking, however I'm not aware of a way to run Android Studio's XML formatter from the command line.

File: app/src/main/res/layout/player_notification.xml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (Stypox) [inline app/src/main/res/layout/player_notification.xml:2]: Oh, seems like I forgot to remove the old notification layout

File: app/src/main/res/layout/player_notification_expanded.xml [hunk 2]
```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout
-    xmlns:android="http://schemas.android.com/apk/res/android"
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (Stypox) [inline app/src/main/res/layout/player_notification_expanded.xml:2]: Also here, could you remove them while you are at it?

**Review by Stypox — CHANGES_REQUESTED** [review]
Can we just assume `Ctrl+Alt+L` does his job correctly and merge directly? ;-)
345 files are too many to check ;-)

Anyway, I took a brief look and there does not seem to be any strange things. I'm not sure if we would need to do some deeper analysis or we can just trust AS. Maybe we can so something like [this](https://stackoverflow.com/questions/3978068/how-to-check-if-the-two-xml-files-are-equivalent-with-python)?

**Comment by wb9688** [comment]
@Stypox: It's just some XML, I'd trust Android Studio for that. Also from taking a quick look at what it did (for the first about 100 files).

**Comment by wb9688** [comment]
Fwiw the source code of Android Studio's XML formatter is at https://android.googlesource.com/platform/tools/base/+/refs/tags/studio-4.0.0/sdk-common/src/main/java/com/android/ide/common/xml/

**Review by Stypox — DISMISSED** [review]
Ok, then @TobiGr merge

**Comment by TacoTheDank** [comment]
You have no idea how happy this makes me.

**Comment by opusforlife2** [comment]
@TacoTheDank You do seem like the kind of person who likes a well-kept codebase. 🤭

**Comment by Stypox** [comment]
For some reason the Synk vulnerability checks are failing, but I don't have the rights to access the results and find out why

**Review by TobiGr — APPROVED** [review]

**Comment by TobiGr** [comment]
snyk is also failing for other PRs. There are no dependency changes in this PR, so this should be good to merge.

**Pull request merged**